src/components/table/examples/table-selectable-rows.tsx

```diff
@@ -0,0 +1,61 @@
+import { Component, h, State } from '@stencil/core';
+import { Column } from '../table.types';
+import { persons, Person } from './persons';
+
+/**
+ * Select rows
+ *
+ * @link persons.ts
+ */
+@Component({
+    tag: 'limel-example-table-selectable-rows',
+    styleUrl: 'table.scss',
+    shadow: true,
+})
+export class TableExampleSelectableRows {
+    @State()
+    private activeRow: Person;
+    @State()
+    private selection: Person[] = [persons[0], persons[2]];
+
+    private tableData: Person[] = persons;
+    private columns: Column[] = [
+        { title: 'Name', field: 'name' },
+        { title: 'Age', field: 'age', horizontalAlign: 'right' },
+        { title: 'Kind', field: 'kind' },
+        { title: 'Height', field: 'height', horizontalAlign: 'right' },
+        { title: 'Stamina', field: 'stamina' },
+        { title: 'Place of Birth', field: 'placeOfBirth' },
+        { title: 'Sign', field: 'sign' },
+        { title: 'Date of Birth', field: 'dateOfBirth' },
+        { title: 'Role', field: 'role' },
+    ];
+
+    public render() {
+        return [
+            <limel-table
+                data={this.tableData}
+                activeRow={this.activeRow}
```

Comment:
Currently in the demo, pressing a row (creating an `activeRow`) will deselect all selected rows. @LucyChyzhova and I think it shouldn't be like that, as explained in the comment above. Looking at Lime Go's interaction, it feels better to keep the active row active and keep the selections. These two interactions shouldn't interfere. 


https://user-images.githubusercontent.com/35954987/149308629-905fcdf6-24fc-40e7-a924-80fe519b33dd.mov

Reply:
Came here to say exactly this 😄

This becomes extra annoying when you just accidentally miss a checkbox 😣

Uploading Screen Recording 2022-01-13 at 11.11.51.mov…

Reply:
Is there any practical reason `activeRow` has to be touched at all by this functionality? Or is that just because you wanted to highlight the selected rows in green, and then `activeRow` became a conflicting thing?

It looks to me like we could just leave `activeRow` alone if we don't make the selected rows green, and we don't want to make them green anyway 🙂

Reply:
Sorry that I'm butting in so much, but I'm just setting out on adding keyboard navigation for setting `activeRow`, so if the checkbox functionality didn't need to touch the `activeRow` functionality at all, that would be pretty great for me, I think 😄

Reply:
> Yes, I forgot to mention this regarding the suggestion of making our own row selection column/ formatter.
It's not only for being able to use a the same style as the limel-checkbox but also to make sure that if you click anywhere it should just use the checkbox select behavior.

Hmm… 🤔 So, you're saying that if the table has checkboxes, clicking anywhere on a row would check the checkbox for that row? Or do you just mean that clicking anywhere in the _cell_ that holds the checkbox should select the checkbox?

Reply:
> Is there any practical reason `activeRow` has to be touched at all by this functionality? Or is that just because you wanted to highlight the selected rows in green, and then `activeRow` became a conflicting thing?
> 
> It looks to me like we could just leave `activeRow` alone if we don't make the selected rows green, and we don't want to make them green anyway 🙂

Sorry for the late replies. This is fixed and the row selection won't touch the activeRow nor active css class since we decided to keep them separate.

Reply:
> > Yes, I forgot to mention this regarding the suggestion of making our own row selection column/ formatter.
> > It's not only for being able to use a the same style as the limel-checkbox but also to make sure that if you click anywhere it should just use the checkbox select behavior.
> 
> Hmm… 🤔 So, you're saying that if the table has checkboxes, clicking anywhere on a row would check the checkbox for that row? Or do you just mean that clicking anywhere in the _cell_ that holds the checkbox should select the checkbox?

Sorry for the vague reply there. I was referring to the cell. That's also why I removed the user interaction with resizing the checkbox cell. 

The current implementation means that using `<limel-table selectable={true}/>` will add a frozen, non-resizable, checkbox column that will have a click handler on the cell that toggles the selection of the row the cell belongs to. Clicking anywhere in that cell, on the checkbox rendered inside it or in the space between the cell border and the checkbox inside it, will toggle the row's selection.

Using Tabulator's default provided shift-click in any cell to extend the selection. We don't want that because it would interfere with shift-click (to open a new window) in cells providing a link (to a relation for instance).
There are many small details in behavior of Tabulator's own row selection that are unwanted.